### DISCUSSION TITLE
How to stop a running process?

### DISCUSSION THREAD

**Discussion by kenherring-rkt** [vscode-abl/vscode-abl#45] [Q&A] [answered]
Let's say I start the following process that gets stuck in an infinite loop.  How do I stop it?  Obviously I can go to the Task Manager, but a simpler way from VSCode would be nice.

```progress
repeat:
  message 100.
end.
```

**Comment by gquerret** [comment, marked as answer]
There's unfortunately currently no way to stop those processes from VSCode
